champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/WindBlade.java

```diff
@@ -30,15 +30,24 @@
 import org.bukkit.event.EventPriority;
 import org.bukkit.event.Listener;
 import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.event.entity.PlayerDeathEvent;
+import org.bukkit.event.player.PlayerQuitEvent;
 import org.bukkit.inventory.meta.ItemMeta;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.UUID;
+import java.util.HashSet;
+import java.util.Set;
 
 @Singleton
 @BPvPListener
 public class WindBlade extends ChannelWeapon implements InteractWeapon, LegendaryWeapon, Listener {
 
+    private static final String ABILITY_NAME = "Flight";
+
+    private final Set<UUID> activeUsageNotifications = new HashSet<>();
```

Comment:
instead of creating this set for every ChannelWeapon, just created a protected set in ChannelWeapon itself